contracts/controller/lib.rs

```diff
@@ -1,3 +1,10 @@
+// Copyright 2023 Asynmatrix Pte. Ltd.
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
```

Comment:
So from my experience this should match the `NOTICE` that you have (see [here](https://spdx.org/licenses/Apache-2.0.html)). So something like: 

```suggestion
// Copyright 2023 Asynmatrix Pte. Ltd.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License. 
```

Your version may also be acceptable, so it's up to you to pick.

Reply:
Updated. thank you, @HCastano